kibana-reports/tsconfig.json

```diff
@@ -9,12 +9,14 @@
     "module": "commonjs",
     "outDir": "./target",
     "noEmit": true,
-    "types": ["node", "jest", "react"],
```

Comment:
Are these not needed? I thought this line had to be included

Reply:
Deleting this line will get rid of the type error on jest related types, such as `describe()`, `test`, `jest.fn()`. I am not sure about other negative affects, any idea?

Reply:
Do tests pass when you run `yarn test --coverage`? Aside from the ones that haven't been updated.

Reply:
![image](https://user-images.githubusercontent.com/32652829/97495172-95f04280-1924-11eb-8c64-4764446fa721.png)

Reply:
I think there are existing snapshot tests that are not fixed yet

Reply:
Ok- should be fine then. It'll be an easy one-line fix if it does end up breaking anything, anyway 👍